### ISSUE TITLE
Dropout in sentence embedder

### ISSUE THREAD

**Issue by hugours** [facebookresearch/LASER#43]
Is it normal that dropout is applied during the forward inference pass for the sentence embedding?

**Comment by avidale** [comment]
Yes, this is normal. 
In the evaluation mode, Dropout just applies the identity function to its inputs (see the PyTorch [documentation](https://pytorch.org/docs/stable/generated/torch.nn.Dropout.html)).
